### PR TITLE
Fix SD null year_of_birth

### PR DESCRIPTION
**Pull request by melissachang**
In source data, some people have null year_of_birth. This caused the cohort preview graphs to not load:

image

Backend error:

```
2023-01-19 15:47:01.207  ERROR 56142 --- [nio-8080-exec-7] b.t.c.e.AbstractGlobalExceptionHandler   : Global exception handler: cause: java.lang.NullPointerException,

java.lang.NullPointerException: null
at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:880)
at com.google.cloud.bigquery.FieldValue.getStringValue(FieldValue.java:109)
at com.google.cloud.bigquery.FieldValue.getLongValue(FieldValue.java:139)
at bio.terra.tanagra.query.bigquery.BigQueryCellValue.getLiteral(BigQueryCellValue.java:63)
at bio.terra.tanagra.service.instances.EntityInstanceCount.fromRowResult(EntityInstanceCount.java:41)
at bio.terra.tanagra.service.QuerysService.runInstanceCountsQuery(QuerysService.java:294)
at bio.terra.tanagra.app.controller.InstancesV2ApiController.countInstances(InstancesV2ApiController.java:228)
```

I used sqlFunctionWrapper to replace `null` with `0` in the index. Now graphs load:

image

### CODE REVIEW

File: .run/Main.run.xml [hunk 1]
```diff
@@ -11,13 +11,13 @@
       <env name="TANAGRA_DB_INITIALIZE_ON_START" value="false" />
       <env name="TANAGRA_DB_USERNAME" value="dbuser" />
       <env name="TANAGRA_DB_PASSWORD" value="dbpwd" />
-      <env name="TANAGRA_UNDERLAY_FILES" value="broad/aou_synthetic/expanded/aou_synthetic.json,broad/cms_synpuf/expanded/cms_synpuf.json" />
+      <env name="TANAGRA_UNDERLAY_FILES" value="broad/aou_synthetic/expanded/aou_synthetic.json,broad/cms_synpuf/expanded/cms_synpuf.json,verily/sdd/expanded/sdd.json" />
```
Comment (marikomedlock) [inline .run/Main.run.xml:14]: Maybe only set this to broad underlays, so in theory anyway, a new developer with broad creds could use this config file.

File: .run/Main.run.xml [hunk 2]
```diff
@@ -11,13 +11,13 @@
...
       <env name="TANAGRA_FEATURE_ARTIFACT_STORAGE_ENABLED" value="true" />
       <env name="TANAGRA_AUTH_IAP_GKE_JWT" value="false" />
-      <env name="GOOGLE_APPLICATION_CREDENTIALS" value="$PROJECT_DIR$/rendered/tanagra_sa.json" />
+      <env name="GOOGLE_APPLICATION_CREDENTIALS" value="$PROJECT_DIR$/../Downloads/verily-tanagra-dev-d9842d6d2d76.json" />
```
Comment (marikomedlock) [inline .run/Main.run.xml:17]: nit: swap local path with rendered one (the one generated by the `pull_credentials.sh` script)

File: service/src/main/resources/config/verily/sdd/expanded/entity/person.json [hunk 3]
```diff
@@ -123,8 +123,8 @@
     "dataType" : "INT64",
     "displayHint" : {
       "type" : "RANGE",
-      "minVal" : 1799.0,
-      "maxVal" : 2993.0
+      "minVal" : 1.0,
+      "maxVal" : 5227737.0
```
Comment (marikomedlock) [inline service/src/main/resources/config/verily/sdd/expanded/entity/person.json:127]: This is interesting -- I would have expected the min to be `0.0` and the max to be unchanged. Are these the right values in the tables?

File: .run/Main.run.xml (hunk 1 above)
Comment (melissachang) [reply .run/Main.run.xml]: Reverted file

File: .run/Main.run.xml (hunk 2 above)
Comment (melissachang) [reply .run/Main.run.xml]: Reverted file

File: service/src/main/resources/config/verily/sdd/expanded/entity/person.json (hunk 3 above)
Comment (melissachang) [reply service/src/main/resources/config/verily/sdd/expanded/entity/person.json]: I re-expanded and these are correct now (0 - 2993). Not sure what happened earlier.

**Review by marikomedlock — APPROVED** [review]

**Pull request merged**